__tests__/app.test.js

```diff
@@ -117,3 +120,54 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: returns an array of all comments for a given article with the correct properties", () => {
+    return request(app)
+      .get("/api/articles/5/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            article_id: expect.any(Number),
+          });
+        });
+      });
+  });
+
+  test("200: comments should return with the most recent comments first, if article_id has comments", () => {
+    return request(app)
+      .get("/api/articles/1/comments?sort_by=created_at&order=desc")
```

Comment:
We don't need to pass a query in yet. The comments should come back sorted by created_at in descending order as default when we go to `/api/articles/1/comments`